vault/resource_userpass_user.go

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+)
+
+func userpassUserResource() *schema.Resource {
+	fields := map[string]*schema.Schema{
+		"backend": {
```

Comment:
Could we update these to use the constants in our `internal/consts/consts.go` package? We should already have constants for all the fields being used here. This [file](https://github.com/hashicorp/terraform-provider-vault/blob/main/vault/resource_ldap_secret_backend_static_role.go) can be used as an example of how constants are used for every schema field :)

Reply:
Also, we are trying to do away w/ the terminology `backend`. Could we update this to be `consts.FieldMount` instead?